packages/text-bitmap/src/BitmapText.ts

```diff
@@ -208,7 +208,7 @@ export class BitmapText extends Container
         this._tint = tint;
         this._fontName = fontName;
         this._fontSize = fontSize || BitmapFont.available[fontName].size;
-        this._text = text;
+        this._text = String(text === null || text === undefined ? '' : text);
```

Comment:
Why not this instead?
```suggestion
        this.text = text;
```

Reply:
that would excecute the following

```ts
        text = String(text === null || text === undefined ? '' : text);

        if (this._text === text)
        {
            return;
        }
        this._text = text;
        this.dirty = true;
```

at first glance the text get's turned into an empty string first, compared against undefined and then stored. That wouldn't be an issue. I just wasn't sure it was a good idea to tie the dirty flag (and any other future change to the setter) to the constructor.

However it can totally be done if you think it is not an issue 👍

Reply:
`dirty` is also set to true in the constructor, so I don't think that's an issue